Source/WebKit/WebProcess/WebProcess.h

```diff
@@ -916,6 +916,9 @@ class WebProcess final : public AuxiliaryProcess {
 #if ENABLE(NOTIFY_BLOCKING)
     HashMap<String, int> m_notifyTokens;
 #endif
+#if ENABLE(LAUNCHSERVICES_SANDBOX_EXTENSION_BLOCKING)
+    std::optional<String> m_pendingDisplayName;
```

Comment:
We normally never use `std::optional` with `String` since String has a null state.